dashboard/app/models/levels/bubble_choice.rb

```diff
@@ -141,6 +152,18 @@ def summarize_sublevels(script_level: nil, user_id: nil)
         level_info[:status] = SharedConstants::LEVEL_STATUS.not_tried
       end
 
+      if should_localize
+        set_unless_nil level_info,
+          :display_name,
+          I18n.t(level.name, scope: [:data, :display_name], default: nil, smart: true)
+        set_unless_nil level_info,
+          :short_instructions,
+          I18n.t(level.name, scope: [:data, :short_instructions], default: nil, smart: true)
+        set_unless_nil level_info,
+          :long_instructions,
+          I18n.t(level.name, scope: [:data, :long_instructions], default: nil, smart: true)
```

Comment:
any reason not to just do something like

```suggestion
        %i[display_name short_instructions_log_instructions].each do |property|
          localized_value = I18n.t(level.name, scope: [:data, property], default: nil, smart: true)
          level_info[property] = localized_value unless localized_value.nil?
        end
```

Reply:
that looks much better!